docs/pages/versions/unversioned/sdk/video.mdx

```diff
@@ -139,6 +139,31 @@ const styles = StyleSheet.create({
 
 </SnackInline>
 
+## Playing local media from the assets directory
+
+`expo-video` does not support playing media directly from a result of calling the `require` function. First, the video has to be loaded using `expo-asset`:
```

Comment:
```suggestion
`expo-video` does not support playing media directly from a result of calling the `require` function. Use `expo-asset` to load the video before using it as the video source:
```

Since there's only once actionable step in the last sentence, let's re-phrase it and also, to add the context about what to do after loading the video with `expo-asset` (that is, to load the video as the source).